Src/WitsmlExplorer.Api/Services/CredentialsService.cs

```diff
@@ -51,7 +49,22 @@ public async Task<string> ProtectBasicAuthorization(string headerValue)
         {
             ServerCredentials credentials = GetBasicCredentialsFromHeaderValue(headerValue);
             await VerifyCredentials(credentials);
-            return Encrypt(credentials.Password);
+            return Encrypt($"{credentials.UserId}:{credentials.Password}");
+        }
+
+        public async Task<ServerCredentials> GetCredentialsCookieFirst(IEssentialHeaders headers, string server)
+        {
+            ServerCredentials result;
+            if (headers.HasCookieCredentials(server))
+            {
+                var cred = Decrypt(headers.GetCookie(server)).Split(":");
```

Comment:
Pretty much a non-issue but I get "Use explicit type instead of 'var'" here. Autoformatting being sluggish?

Reply:
yes, did not get that here? but true, it should,  I will replace it